modules/core/tests/server/core.server.config.tests.js

```diff
@@ -105,7 +105,7 @@ describe('Configuration Tests:', function () {
               }
 
               // The user we just created should be exposed
-              res.text.should.containEql('user = {"_id":"' + userId + '",');
+              res.text.should.match(new RegExp('user = \\{.*"_id":"' + userId + '"'));
```

Comment:
Did some new fields appear before `_id` or did the order of fields change?

Reply:
Order of the fields seems to have changed.